plugins/woocommerce/templates/order/order-details.php

```diff
@@ -82,6 +87,37 @@
 			?>
 		</tbody>
 
+		<?php if ( ! empty( $actions ) ) : ?>
+		<tfoot>
+			<tr>
+				<th style="vertical-align:middle">Actions: </th>
+				<td>
+						<?php
+						$actions     = wc_get_account_orders_actions( $order );
+						$actions     = array_filter(
+							$actions,
+							function ( $action ) {
+								return 'View' !== $action['name'];
+							}
+						);
+						$num_actions = count( $actions );
+						$i           = 0;
+						foreach ( $actions as $key => $action ) { // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+							if ( empty( $action['aria-label'] ) ) {
+								// Generate the aria-label based on the action name.
+								/* translators: %1$s Action name, %2$s Order number. */
+								$action_aria_label = sprintf( __( '%1$s order number %2$s', 'woocommerce' ), $action['name'], $order->get_order_number() );
+							} else {
+								$action_aria_label = $action['aria-label'];
+							}
+								echo '<a href="' . esc_url( $action['url'] ) . '" class="woocommerce-button' . esc_attr( $wp_button_class ) . ' button ' . sanitize_html_class( $key ) . '" aria-label="' . esc_attr( $action_aria_label ) . '" style="' . ( ++$i === $num_actions ? '' : 'margin-right: 10px;' ) . '">' . esc_html( $action['name'] ) . '</a>';
```

Comment:
I'm getting a an error from this `$wp_button_class` in 2024 theme

```
Warning: Undefined variable $wp_button_class in woocommerce/templates/order/order-details.php on line 113
```

Reply:
Also, margin should be done via CSS with a new classname, examples in https://github.com/woocommerce/woocommerce//blob/8716d979964e863034125acd0204e89333b9e393/plugins/woocommerce/client/legacy/css/woocommerce.scss#L1364

Reply:
Missing class is generated like this:

```php
$wp_button_class = wc_wp_theme_get_element_class_name( 'button' ) ? ' ' . wc_wp_theme_get_element_class_name( 'button' ) : '';
```